File: CursoJava/Exer/src/Outros/ListaAlgoritimo/Algoritimo2/Exer06.java
/*
6) Escreva um algoritmo que lê dois valores booleanos (lógicos)
e então determina se ambos são VERDADEIROS ou FALSOS.
 */

package Outros.ListaAlgoritimo.Algoritimo2;

import java.util.Scanner;

public class Exer06 {
    public static void main(String[] args) {
        boolean a, b;
        Scanner sc = new Scanner(System.in);

        System.out.print("Digite o primeiro valor booleano (true/false): ");
        a = sc.nextBoolean();

        System.out.print("Digite o segundo valor booleano (true/false): ");
        b = sc.nextBoolean();

        if (a && b) {
            System.out.println("Ambos são verdadeiros.");
        } else if (!a && !b) {
            System.out.println("Ambos são falsos.");
        } else {
            System.out.println("Um dos valores é falso.");
        }

        sc.close();
    }
}

File: CursoPython3/exer/listaAlgoritimos/algoritimo2/exer06.py
"""
6) Escreva um algoritmo que lê dois valores booleanos (lógicos) e então determina se ambos são
VERDADEIROS ou FALSOS.
"""

a = bool(input("Digite o valor de A: "))
b = bool(input("Digite o valor de B: "))

if a == True and b == True:
    print("Ambos são verdadeiros.")
elif a == False and b == False:
    print("Ambos são falsos.")
else:
    print("Um dos valores é falso.")
